specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/examples/JobsListSecrets.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "subscriptionId": "3c66da21-607e-49b4-8bdf-f25fbb8f705f",
+    "resourceGroupName": "SdkRg2508",
+    "jobName": "sdktest5497",
+    "api-version": "2020-04-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+      }
```

Comment:
why the body is empty?

Reply:
Properties jobName and jobSecrets were giving errors when I tried to validate the spec and the examples. Since Jobs_ListCredentials operation didn't have any required response parameter, I kept the body empty.

Reply:
I added the response body. Thank you.